spec/koans.js

```diff
@@ -85,54 +93,56 @@ describe('`string.includes()` finds string within another string. ', () => {
 
   describe('find a single character', function() {
     it('in a three char string', function() {
-      /* const searchString = ???? */
-      //expect('xyz'.includes(searchString)).toBe(true);
+       const searchString = 'xyz'
```

Comment:
Este debe de ser in single character es decir uno de los tres de la palabra en la que tiene que buscarse, 'x', 'y' o 'z'